### PR TITLE
fix error on declaring docker_container with port_mapping

### PR DESCRIPTION
**Pull request by mike0sv**

### CODE REVIEW

**Review by aguschin — COMMENTED** [review]
Great! Tried out this branch.
```
$ mlem declare deployment docker_container docker2 --port_mapping.9000 8080
```
that works here, and that don't work in `main`.

But I see two arguments in `--help`:
```
│ --port_mapping                      DICT  Expose ports [default: __NOT_SET__]                                                                        │
│ --port_mapping.key                  INT   Expose ports [default: __NOT_SET__]                                                                        │
```
the example I posted is about using the 2nd option. What about the 1st option? Wasn't able to use it somehow. Does it works? If yes, could you please post an example? If not, let's simply remove it?

File: tests/cli/test_declare.py [hunk 1]
```diff
@@ -342,6 +342,68 @@ class AllowNoneField(_MockBuilder):
...
+
+
+# TODO
+# all_test_params.extend(
+#     (
+#         pytest.param(
+#             DoubleDictIntKey(field={10:{10:10}}), "--field.10.10 10", id="double_dict_int_key"
+#         ),
+#     )
+# )
```
[the remark this answers is not in the document]
Comment (mike0sv) [reply tests/cli/test_declare.py:385]: Can we do both?

**Comment by mike0sv** [comment]
@aguschin This is not new actually. I think I left it there to highlight that this field is dict. But that's another issue

File: tests/cli/test_declare.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (aguschin) [reply tests/cli/test_declare.py:385]: Please fix if that doesn't look correct https://github.com/iterative/mlem/issues/483

[the remark this answers is not in the document]
Comment (aguschin) [reply tests/cli/test_declare.py:385]:
```suggestion
# TODO https://github.com/iterative/mlem/issues/483
# all_test_params.extend(
#     (
#         pytest.param(
#             DoubleDictIntKey(field={10:{10:10}}), "--field.10.10 10", id="double_dict_int_key"
#         ),
#     )
# )
```

**Review by aguschin — APPROVED** [review]

**Pull request merged**